src/pybind/rbd.pyx

```diff
@@ -0,0 +1,1431 @@
+# cython: embedsignature=True
+"""
+This module is a thin wrapper around librbd.
+
+It currently provides all the synchronous methods of librbd that do
+not use callbacks.
+
+Error codes from librbd are turned into exceptions that subclass
+:class:`Error`. Almost all methods may raise :class:`Error`
+(the base class of all rbd exceptions), :class:`PermissionError`
+and :class:`IOError`, in addition to those documented for the
+method.
+"""
+# Copyright 2011 Josh Durgin
+# Copyright 2015 Hector Martin <marcan@marcan.st>
+
+from cpython cimport PyObject, ref, exc
+from libc cimport errno
+from libc.stdint cimport *
```

Comment:
Is there value in explicitly listing all the integer types in use? Currently it just seems to be two (uint8_t and uint64_t), but that could change as new librbd methods are added. Keep in mind that this is just a Cython definition file that is only relevant at compile time, so there is no runtime or binary size overhead, nor any difference as far as module users are concerned (nothing is exposed to the Python side); it's the moral equivalent of `#include` in C.